src/Program.cs

```diff
@@ -7,39 +7,62 @@ class Program
-        }
+            double frontPriceMeterSquared = PanelPriceMeterSquared(frontThickness);
+            double sidesPriceMeterSquared = PanelPriceMeterSquared(sidesThickness);
+            double bottomPriceMeterSquared = PanelPriceMeterSquared(bottomThickness);
 
-        static void PrintResult(int height, int width, int depth, int totalGlassArea, double waterVolume, double sideThickness, double frontAndBackThickness, double bottomThickness)
-        {
-            Console.WriteLine($"\n\n\nDimensions of the aquarium are {width} mm (base) x {depth} mm (base) x {height} mm (height).\n");
-            Console.WriteLine($"Total glass area is {Math.Round(((double)(totalGlassArea / 1000000)), 2)} m\u00b2.\n");
-            Console.WriteLine($"Maximum water volume is {waterVolume / 1000000} litres.\n");
-            Console.WriteLine($"Side panels thickness is {Math.Round(sideThickness, 2)} mm.");
-            Console.WriteLine($"Front and back panels thickness is {Math.Round(frontAndBackThickness, 2)} mm.");
-            Console.WriteLine($"Bottom panel thickness is {Math.Round(bottomThickness, 2)} mm.\n");
+            double frontTotalPrice = frontPriceMeterSquared * frontPanelsArea;
+            double sideTotalPrice = sidesPriceMeterSquared * sidePanelsArea;
+            double bottomTotalPrice = bottomPriceMeterSquared * bottomPanelArea;
 
-            Console.ReadKey();
-        }
+            double totalPanelPrice = totalPrice(frontTotalPrice, sideTotalPrice, bottomTotalPrice);                
 
-        static int GetDimension(string dimensionType)        
+            Console.WriteLine($"\n\n\nDimensions of the aquarium are {width} mm (base) x {depth} mm (base) x {height} mm (height).\n");
+            Console.WriteLine($"Total glass area is {Math.Round(totalGlassArea, 2)} m\u00b2.\n");
+            Console.WriteLine($"Maximum water volume is {volume} litres.\n");
+            Console.WriteLine($"Side panels minimum thickness is {Math.Round(sidesMinimumThickness, 2)} mm.");
+            Console.WriteLine($"Front and back panels minimum thickness is {Math.Round(frontMinimumThickness, 2)} mm.");
+            Console.WriteLine($"Bottom panel minimum thickness is {Math.Round(bottomMinimumThickness, 2)} mm.\n\n\n");
+            Console.WriteLine("Panels needed:");
+            Console.WriteLine("\tFront and back panels:"); 
+            Console.WriteLine($"\t\t2 panels with dimensions {height} mm x {width} mm x {frontThickness} mm");
+            Console.WriteLine($"\t\tTotal area {frontPanelsArea * 2} m\u00b2");
+            Console.WriteLine($"\t\tprice per m\u00b2 {frontPriceMeterSquared} EUR"); 
+            Console.WriteLine($"\t\t\tTotal price {frontTotalPrice * 2}\n");
+            Console.WriteLine("\tSide panels:"); 
+            Console.WriteLine($"\t\t2 panels with dimensions {height} mm x {width} mm x {sidesThickness} mm");
```

Comment:
Ovo je veliki blok, prilično nepregledan i veoma podložan greškama, kao što se može videti na primeru u ovoj liniji, ponovo se štamapju iste dimenzije kao i za front/back panele. To je verovatno posledica kopiranja koda.
Čak i ako nije kopiran, ovde ima dosta "duplog koda" koji možda nije identičan po sadržaju ali je ekvivalentan ili isti po nameni. To je rasadnik bagova, kako sada, tako i kasnije kad bude trebalo nešto ispraviti.